src/Microsoft.Extensions.Testing.Abstractions/DIA/DiaDataSource.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
I don't want to ship an SDK on behalf of the debugger folks. Let's make it clear these are specific to our test abstractions.